### PR TITLE
HBASE-23621 Reduced the number of Checkstyle violations in tests of hbase-common

### PR DESCRIPTION
**Pull request by HorizonNet**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   7m 19s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 10 new or modified test files.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   7m 38s |  master passed  |
| +1 :green_heart: |  compile  |   0m 25s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 33s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m 41s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 27s |  master passed  |
| +0 :ok: |  spotbugs  |   0m 57s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 55s |  master passed  |
||| _ Patch Compile Tests _ |
| -1 :x: |  mvninstall  |   2m 51s |  root in the patch failed.  |
| +1 :green_heart: |  compile  |   0m 24s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 24s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 28s |  hbase-common: The patch generated 0 new + 0 unchanged - 40 fixed = 0 total (was 40)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  shadedjars  |   3m 45s |  patch has 15 errors when building our shaded downstream artifacts.  |
| -1 :x: |  hadoopcheck  |   1m 49s |  The patch causes 15 errors with Hadoop v2.8.5.  |
| -1 :x: |  hadoopcheck  |   3m 39s |  The patch causes 15 errors with Hadoop v2.9.2.  |
| -1 :x: |  hadoopcheck  |   5m 35s |  The patch causes 15 errors with Hadoop v3.1.2.  |
| +1 :green_heart: |  javadoc  |   0m 24s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   1m  1s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   3m 22s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 12s |  The patch does not generate ASF License warnings.  |
|  |   |  42m 44s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/967 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux a3f646f13926 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-967/out/precommit/personality/provided.sh |
| git revision | master / e9b998012f |
| Default Java | 1.8.0_181 |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/1/artifact/out/patch-mvninstall-root.txt |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/1/artifact/out/patch-shadedjars.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/1/artifact/out/patch-javac-2.8.5.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/1/artifact/out/patch-javac-2.9.2.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/1/artifact/out/patch-javac-3.1.2.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/1/testReport/ |
| Max. process+thread count | 252 (vs. ulimit of 10000) |
| modules | C: hbase-common U: hbase-common |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  7s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 11 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 46s |  master passed  |
| +1 :green_heart: |  compile  |   0m 41s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 40s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m  3s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 36s |  master passed  |
| +0 :ok: |  spotbugs  |   0m 33s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   1m 22s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 35s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 39s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 39s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 27s |  hbase-common: The patch generated 0 new + 0 unchanged - 40 fixed = 0 total (was 40)  |
| +1 :green_heart: |  checkstyle  |   0m 12s |  The patch passed checkstyle in hbase-zookeeper  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   5m  0s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  17m 18s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| +1 :green_heart: |  javadoc  |   0m 34s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   1m 35s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   3m  7s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 48s |  hbase-zookeeper in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 22s |  The patch does not generate ASF License warnings.  |
|  |   |  58m 34s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/967 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux c701464b0781 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-967/out/precommit/personality/provided.sh |
| git revision | master / e9b998012f |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/2/testReport/ |
| Max. process+thread count | 292 (vs. ulimit of 10000) |
| modules | C: hbase-common hbase-zookeeper U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-967/2/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-common/src/test/java/org/apache/hadoop/hbase/util/RedundantKVGenerator.java [hunk 1]
```diff
@@ -469,95 +463,9 @@ public static ByteBuffer convertKvToByteBuffer(List<KeyValue> keyValues,
...
-
-  public RedundantKVGenerator setBaseTimestampDivide(int baseTimestampDivide) {
-    this.baseTimestampDivide = baseTimestampDivide;
-    return this;
-  }
-
-  public RedundantKVGenerator setTimestampDiffSize(int timestampDiffSize) {
-    this.timestampDiffSize = timestampDiffSize;
-    return this;
-  }
```
Comment (saintstack) [inline hbase-common/src/test/java/org/apache/hadoop/hbase/util/RedundantKVGenerator.java]: Not used?

Comment (HorizonNet) [reply hbase-common/src/test/java/org/apache/hadoop/hbase/util/RedundantKVGenerator.java]: Doesn't seem to be the case. Didn't find any usages.

Comment (saintstack) [reply hbase-common/src/test/java/org/apache/hadoop/hbase/util/RedundantKVGenerator.java]: Ok. Good.

**Pull request closed without merging**